Scripts/create_deb_packages.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# create binary *.deb packages
+
+#set -x # DEBUG
+
+mkdir -p build
+cd build
+cmake \
+    -DRDK_INSTALL_INTREE=OFF \
+    -DRDK_BUILD_INCHI_SUPPORT=ON \
+    -DRDK_BUILD_AVALON_SUPPORT=ON \
+    -DRDK_BUILD_PYTHON_WRAPPERS=ON \
+    -DCMAKE_INSTALL_PREFIX=/usr \
+    ../
+nprocs=`getconf _NPROCESSORS_ONLN`
+make -j $nprocs
+cpack -G DEB
+
+# # to install all necessary dependencies on Ubuntu
```

Comment:
I'm not sure which version of Ubuntu this is intended for, but it doesn't look like a complete list of depedencies to me. Here's what I ended up using to do a python3 build on 18.04:
```
apt install -y \
  curl \
  wget \
  libboost-all-dev \
  cmake \
  git \
  g++ \
  libeigen3-dev \
  python3 \
  libpython3-all-dev \
  python3-numpy \
  python3-pip \
  python3-pil \
  python3-six \
  python3-pandas
```